generators/app/templates/src/utils/array.js

```diff
@@ -6,15 +6,8 @@ import Immutable from 'seamless-immutable';
  * stringArrayToObject(['A', 'B', 'C']) // { A: 'A', B: 'B', C: 'C' }
  */
 export function stringArrayToObject(actionsArray, namespace = '') {
-  if (
-    actionsArray.some(
-      actionName => !actionName || typeof actionName !== 'string'
-    )
-  ) {
+  if (actionsArray.some(actionName => !actionName || typeof actionName !== 'string')) {
     throw new Error('Action names must be strings and must not be empty');
```

Comment:
I'd delete this in favor of r-r `createTypes`

Reply:
same